std/log/mod_test.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+const { test } = Deno;
+import { Logger } from "./logger.ts";
+import { assert } from "../testing/asserts.ts";
+import { getLogger } from "./mod.ts";
+
+let logger: Logger | null = null;
+try {
```

Comment:
Done @bartlomieju it was a bit tricky because when the test is executed the logger was already setup.